### ISSUE TITLE
refactoring, mostly to remove utils.py

### ISSUE THREAD

**Issue by jeff-regier** [prob-ml/bliss#17]
@ismael2395 , here's a list of my refactoring suggestions:

- [x] utils.py: remove paths; get those from command line args with defaults (can just specify root); pass them as needed to functions
- [x] in `celeste/__init__.py` set `device = torch.device("cuda" if use_cuda else "cpu")`
- [x] utils.py: delete FloatTensor / LongTensor; instead, use `torch.tensor(..., dtype=..., device=device)` syntax (can omit dtype for float32)
- [x] utils.py: replace `image_h5_name` and `background_h5_name` variables with hard coded string
- [x] utils.py: `get_is_on_from_n_sources` only used in simulated_dataset.py so move it there
- [x] utils.py: `load_data_params_from_args` is only used in run_sleep_galaxy.py so move it there; same for other functions in utils.py that are only used one place
- [x] delete old notebooks
- [x] delete `get_one_hot_encoding_from_int` and use `torch.nn.functional.one_hot` instead, and the remove unit test of `get_one_hot_encoding_from_int`
- [x] delete `TestUtils` — these aren't going to break
- [x] move sdss_psf.py into datasets/sdss.py -- get the sdss specific stuff to one file
- [x] delete Normalize2d
- [x] delete commented out code in train.py
